pandas/io/spss.py

```diff
@@ -46,6 +47,8 @@ def read_spss(
           DataFrame.
 
         .. versionadded:: 2.0
+    kwargs : dict, optional
```

Comment:
Same as above.

Reply:
<img width="596" alt="image" src="https://github.com/pandas-dev/pandas/assets/20301271/6fc3af79-cb63-49cb-a513-11a5d5ad426f">

Reply:
Can you add the `**` to the parameter name? Otherwise you are documenting this as regular parameter with the name `kwargs`, not the special `**kwargs`.

Reply:
Done.